index.js

```diff
@@ -51,25 +28,62 @@ exports.promptPrefix = '';
 exports.debugFlag = '--debug';
 
 /**
- * Boot mage
+ * Force-configure mage to use cluster: 1 (one master, one process),
+ * and set the list of files and folders to watch for.
+ */
+var config = require('../mage/lib/config');
```

Comment:
Why the `../` before `mage`? Module resolution should find `mage/lib/config` just fine, right?